src/nvim/types.h

```diff
@@ -13,4 +13,5 @@ typedef unsigned char char_u;
 // Can hold one decoded UTF-8 character.
 typedef uint32_t u8char_T;
 
+typedef struct expand expand_T;
```

Comment:
I get a lua error for missing declaration for expand_T with that still if I run 
`TEST_FILE=test/unit/option_spec.lua make unittest `
With plain `make unittest` I get all passed
<code>Error → ./test/unit/helpers.lua @ 118
suite test/unit/option_spec.lua
./test/unit/helpers.lua:118: declaration specifier expected near 'expand_T' at line 42

stack traceback:
        ./test/unit/helpers.lua:118: in function 'cimport'
        test/unit/option_spec.lua:7: in main chunk
</code>